WARNING:root:You are using evm version 1.8.2. The supported version is 1.7.3
WARNING:root:You are using solc version 0.4.21, The latest supported version is 0.4.19
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 0/0
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Money concurrency bug: False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
Traceback (most recent call last):
  File "/oyente/oyente/oyente.py", line 221, in <module>
    main()
  File "/oyente/oyente/oyente.py", line 210, in main
    exit_code = analyze_bytecode()
  File "/oyente/oyente/oyente.py", line 72, in analyze_bytecode
    result, exit_code = symExec.run(disasm_file=inp['disasm_file'])
  File "/oyente/oyente/symExec.py", line 2458, in run
    ret = detect_vulnerabilities()
  File "/oyente/oyente/symExec.py", line 2315, in detect_vulnerabilities
    return results, vulnerability_found()
  File "/oyente/oyente/symExec.py", line 2346, in vulnerability_found
    vulnerabilities = [callstack, money_concurrency, time_dependency, reentrancy]
NameError: global name 'callstack' is not defined
